README.md

```diff
@@ -4,24 +4,39 @@ This is our SOEN 341 project (Winter 2022, ConU, Group K)
 
 ## Objective
 
-The objective of this project is to provide a user an online shopping experience with the opportunity to create a shopping cart based on different categories. 
+The goal of this project is to provide users with an online shopping application that allows them to conveniently shop for their favourite items from an extensive number of options available on the web.
 
-## Project
+## Project Description
+Boreal is a web-based store that offers a large selection of the most influential writings on the market. Users may enjoy the same traditional shopping experience from the comfort of their own homes. The online store will only be accessible to those who have been authenticated by a secure login or sign up process. Users can log in or create an account, browse items, and add them to their shopping cart.Sellers will be able to list their items for sale on the website. This design features a straightforward but effective layout.
+
+## Core Features
+*Items in the shopping cart can be added or removed
```

Comment:
Add a space between the star character and the core features to make the bullet points visible.
Code example: `* Items in...`

Reply:
Fixes implemented. Thank you.